packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/fragment.js

```diff
@@ -9,58 +10,85 @@ import { build_template_literal, build_update } from './utils.js';
  * (e.g. `{a} b {c}`) into a single update function. Along the way it creates
  * corresponding template node references these updates are applied to.
  * @param {SvelteNode[]} nodes
- * @param {(is_text: boolean) => Expression} expression
+ * @param {(is_text: boolean) => Expression} initial
```

Comment:
It's a bit confusing that the parameter is called `is_text` but below it's `check` or `needs_hydration_check` - we should align that

Reply:
yeah i was being indecisive. aligned on `is_text`